### ISSUE TITLE
Question about the initial augmented state

### ISSUE THREAD

**Issue by Frankie123421** [urtrial/neural_ode#4]
Hello, I want to ask about the last element of s0 in the picture of algorithm 1. Why the initial value of at(t1) is -dL/dt1 but not dL/dt1? I don't understand the minus sign here.

**Comment by lbq8942** [comment]
me too,have you solved it?

**Comment by Frankie123421** [comment]
No, and I've escaped from "neural ode" now lol. Sorry about that.

**Comment by lbq8942** [comment]
hahaha, i am still in this trap. Thank you, anyway.&nbsp;




青山
***@***.***



&nbsp;





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "msurtsukov/neural-ode"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月4日(星期五) 下午4:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [msurtsukov/neural-ode] Question about the initial augmented state (#4)





 
No, and I've escaped from "neural ode" now lol. Sorry about that.

**Comment by chrisway613** [comment]
> 

So，Have u solved this problem now?

**Comment by Huaijiu1** [comment]
Hello! Have you solved the problem now?
